From: Richard Acayan <mailingradian@gmail.com>
Date: Fri, 6 Dec 2024 17:02:47 -0500
Subject: [PATCH] media: i2c: imx363: move asm/unaligned.h to
 linux/unaligned.h

In commit 5f60d5f6bbc1 ("move asm/unaligned.h to linux/unaligned.h")
included by Linux 6.12-rc2, the asm/unaligned.h header was moved.
Implement that change in the downstream IMX363 driver to fix compiling.

Signed-off-by: Richard Acayan <mailingradian@gmail.com>
---
 drivers/media/i2c/imx363.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/i2c/imx363.c b/drivers/media/i2c/imx363.c
index fc1fdda..93d6198 100644
--- a/drivers/media/i2c/imx363.c
+++ b/drivers/media/i2c/imx363.c
@@ -8,11 +8,11 @@
 #include <linux/module.h>
 #include <linux/pm_runtime.h>
 #include <linux/regulator/consumer.h>
+#include <linux/unaligned.h>
 #include <media/v4l2-cci.h>
 #include <media/v4l2-ctrls.h>
 #include <media/v4l2-device.h>
 #include <media/v4l2-fwnode.h>
-#include <asm/unaligned.h>
 
 #define IMX363_REG_MODE_SELECT	CCI_REG8(0x0100)
 #define IMX363_MODE_STANDBY		0x00
